ui/src/components/DiffTextArea/index.jsx

```diff
@@ -0,0 +1,203 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { string, func, number } from 'prop-types';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { ReactGhLikeDiff } from 'react-gh-like-diff';
+import 'react-gh-like-diff/lib/diff2html.css';
```

Comment:
Running this locally shows:

```
error: Unable to resolve path to module 'react-gh-like-diff/lib/diff2html.css'
```

Where in the docs do they recommend importing this file?

Reply:
I refer to `react-gh-like-diff` package's [demo](https://codesandbox.io/s/qvx7438xo6) page.
And it is based on this file [diff2html.css](https://github.com/rtfpessoa/diff2html/blob/master/src/ui/css/diff2html.css) from `diff2html` package. Maybe we need to create a new one for our own?
BTW, I did not get the message while running my local, not sure why 🤔

ref: https://github.com/neighborhood999/react-gh-like-diff

Reply:
Removing node_modules and then running `yarn` again fixed the issue for me.